src/selic-entries.spec.ts

```diff
@@ -0,0 +1,5 @@
+import { selicEntries } from "./selic-entries";
+
+test("Testing selicEntries function", async () => {
+    expect(await selicEntries("08/03/2024","08/03/2024")).toBe('[{"data":"08/03/2024","valor":"0.041957"}]')
```

Comment:
Why are we testing a string as the result of a method should return a json?

Reply:
Give it a read on how Jest do assertions: https://jestjs.io/docs/expect

There is one to compare objects that's neat.